tools/WinObjC.Packaging/project.json

```diff
@@ -1,8 +1,7 @@
 ﻿{
   "dependencies": {
     "GitInfo": "1.1.32",
-    "NuGet.Build.Packaging": "0.1.186",
-    "WinObjC.Language": "0.2.160915-*"
```

Comment:
>"WinObjC.Language": "0.2.160915-*" [](start = 4, length = 34)

@memontic-ms This was causing issues for totally clean builds. Since it builds at the same time as Language and the build machine I set to not look at VSO for packages to ensure its locally built ones were used, it couldn't restore this. I couldn't find / remember what this was needed for. Is this ok?